### PR TITLE
UI word list, categories and controller skeleton

### PR DESCRIPTION
**Pull request by NguyenTanDuc**

### CODE REVIEW

**Comment by brokenheart1712** [comment]
I see no problem here.

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
   root                'static_pages#home'
+  resources :words
+  resources :categories
+  resources :lessons, only: [:index, :create, :show, :destroy]
```
Comment (dieunb) [inline config/routes.rb]: use `except` instead of `only`

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
-  root                'static_pages#home'
+  root      'static_pages#home'
+  resources :words
+  resources :categories
+  resources :lessons, expect: [:edit, :update]
```
Comment (dieunb) [inline config/routes.rb]: `except`

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,19 @@
...
+
+50.times do |n|
+  content = "English#{n+1}"
+  Word.create!( content: content )
+end
+
+words = Word.order(:created_at).take(50)
+4.times do |n|
+  content = "Vietnamese"
+  if n==1
```
Comment (dieunb) [inline db/seeds.rb]: Remove
```
+  if n==1 
+    correct = true
+  else
+    correct = false
+  end
```

File: db/seeds.rb [hunk 4]
```diff
@@ -5,3 +5,19 @@
...
+
+words = Word.order(:created_at).take(50)
+4.times do |n|
+  content = "Vietnamese"
+  if n==1 
+    correct = true
+  else
+    correct = false
+  end
+  words.each { |word| word.answers.create!(content: content, correct: correct) }
```
Comment (dieunb) [inline db/seeds.rb]:
```
 words.each {|word| word.answers.create! content: content, correct: n == 1}
```

**Pull request closed without merging**